src/style.css

```
@@ -1 +1,9 @@
-/* styles go here */
+.form-field {
+  margin-bottom: 10px;
+  display: block;
+}
+
+fieldset {
+  padding-bottom: 0;
+  margin-bottom: 20px;
+}
```

[checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md):
- 6th item, use class selectors instead of tag ones
- 9th item, any other styles besides margins, should be browser default.
```suggestion
.your_class_name {
  margin-bottom: 20px;
}
```